package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-plugin-naming-convention",
+  "version": "1.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "neutrino start --use @neutrinojs/node",
+    "build": "neutrino build --use @neutrinojs/node"
```

Comment:
We don't need `--use` flag when we have `.neutrinorc` file in the project root. All information is taken from the file autimatically